src/lib/connectors/repos/billing-volumes.js

```diff
@@ -142,12 +142,12 @@ const updateByBatchId = async (billingBatchId, changes) => {
  * Marks all records as errored by batch ID
  * @param {String} billingBatchId
  */
-const markVolumesAsErrored = async (billingBatchId) => {
+const markVolumesAsErrored = async (billingBatchId, options = {}) => {
```

Comment:
Do you think it's worth changing the default for `options` from `{}` to `{required: false}`?

Reply:
I don't know if we would want to make it the default behaviour. Also, I think that the function call `markVolumesAsErrored(batchId, { require: false })` is clearer than `markVolumesAsErrored(batchId, {})` when providing the second variable. 

I have done this same type of thing with a `isUpdateRequired` boolean flag as the second parameter which defaults to false, but I thought that `options = {}` was most flexible and provided the best readability.